src/js/base/backbone-fetch.js

```diff
@@ -1,56 +1,30 @@
-import { extend, isError } from 'underscore';
+import { extend } from 'underscore';
 import Backbone from 'backbone';
-import baseFetch, { getData } from 'js/base/fetch';
+import baseFetch, { getData, handleError } from 'js/base/fetch';
 
 Backbone.ajax = options => {
-  let isAborting = false;
-  const controller = new AbortController();
-
   options = extend({
     method: options.type,
-    signal: controller.signal,
   }, options);
 
   const fetcher = baseFetch(options.url, options)
-    .then(response => {
-      const promise = getData(response, options.dataType);
-
-      if (response.ok) return promise;
-
-      const error = new Error(response.statusText || response.status || 'Unknown Error');
-      return promise.then(responseData => {
-        error.response = response;
-        error.responseData = responseData;
-        if (options.error) options.error(error);
-        else throw error;
-        return Promise.reject(error);
-      });
-    })
-    .then(responseData => {
-      fetcher.readyState = 'DONE';
+    .then(async response => {
```

Comment:
Logic in the backbone section of the fetch was greatly simplified.  This should look similar to the fetchBy and is generally handling whether to pass back the response or reject the response and data.